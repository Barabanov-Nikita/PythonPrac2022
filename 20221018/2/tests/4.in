:neg x -x
neg 1
:hello "Hi"
hello
:add x y x+y
add 2 3
:mul x y z w x*y*z*w
mul 1 2 3 4
quit "{}/{}"