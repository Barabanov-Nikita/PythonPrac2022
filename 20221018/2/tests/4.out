-1
Hi
5
24
5/9
